geanypy/src/geanypy-editor.c

```diff
@@ -62,7 +62,7 @@ Editor_get_property(Editor *self, const gchar *prop_name)
 		PyObject *py_doc;
 		py_doc = (PyObject *) Document_create_new_from_geany_document(
 									self->editor->document);
-		if (py_doc && py_doc != Py_None)
+		if (!py_doc || py_doc == Py_None)
```

Comment:
I didn't look into this difference, I just picked the kind of condition that made the branch execute when it seemed plausible to me.